lib/src/otx/backend/native/models/detection/heads/dfine_decoder.py

```diff
@@ -987,12 +979,14 @@ class DFINETransformer:
             "feat_channels": [384, 384, 384],
             "reg_scale": 8.0,
             "eval_idx": -1,
-            "eval_spatial_size": [640, 640],
             "activation": nn.SiLU,
         },
     }
 
-    def __new__(cls, model_name: str, num_classes: int) -> DFINETransformerModule:
+    def __new__(
+        cls, model_name: str, num_classes: int, eval_spatial_size: tuple[int, int] | None = None
```

Comment:
Is there any reason why `eval_spatial_size` is optional here? Since it’s always overridden at line 991, why not declare it with a default value in the method signature?